### PR TITLE
Update react-loadable-ssr-addon

### LINKED ISSUE #3003: Project broken on Yarn v2
## 🐛 Bug Report

When on Yarn 2, I get this error when trying to run `yarn start`:

```js
/workspace/Cherry/website/.pnp.js:28332
    throw firstError;
    ^

Error: A package is trying to access another package without the second one being listed as a dependency of the first one

Required package: @babel/runtime (via "@babel/runtime/helpers/interopRequireDefault")
Required by: react-loadable-ssr-addon@virtual:ebd7b9259ecd2093755e637a1fd861d705df40e70f047e72ef912a060f6f8b0d7d8b754ceeb087469fa51421c9654efe3ad7ba9d5fbe846f41f5175c740dcb64#npm:0.2.3 (via /workspace/Cherry/website/.yarn/$$virtual/react-loadable-ssr-addon-virtual-3a33825c47/0/cache/react-loadable-ssr-addon-npm-0.2.3-26d8c8d441-1.zip/node_modules/react-loadable-ssr-addon/lib/)

Require stack:
- /workspace/Cherry/website/.yarn/$$virtual/react-loadable-ssr-addon-virtual-3a33825c47/0/cache/react-loadable-ssr-addon-npm-0.2.3-26d8c8d441-1.zip/node_modules/react-loadable-ssr-addon/lib/index.js
- /workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/lib/commands/build.js
- /workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/lib/index.js
- /workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/bin/docusaurus.js
    at Object.makeError (/workspace/Cherry/website/.pnp.js:20077:24)
    at resolveToUnqualified (/workspace/Cherry/website/.pnp.js:28940:35)
    at resolveRequest (/workspace/Cherry/website/.pnp.js:29017:27)
    at Object.resolveRequest (/workspace/Cherry/website/.pnp.js:29085:26)
    at Function.module_1.Module._resolveFilename (/workspace/Cherry/website/.pnp.js:28315:34)
    at Function.module_1.Module._load (/workspace/Cherry/website/.pnp.js:28200:40)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/workspace/Cherry/website/.yarn/$$virtual/react-loadable-ssr-addon-virtual-3a33825c47/0/cache/react-loadable-ssr-addon-npm-0.2.3-26d8c8d441-1.zip/node_modules/react-loadable-ssr-addon/lib/index.js:3:30)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.module_1.Module._load (/workspace/Cherry/website/.pnp.js:28230:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/lib/commands/build.js:16:52) {
  code: 'MODULE_NOT_FOUND',
  pnpCode: 'UNDECLARED_DEPENDENCY',
  data: {
    request: '@babel/runtime/helpers/interopRequireDefault',
    issuer: '/workspace/Cherry/website/.yarn/$$virtual/react-loadable-ssr-addon-virtual-3a33825c47/0/cache/react-loadable-ssr-addon-npm-0.2.3-26d8c8d441-1.zip/node_modules/react-loadable-ssr-addon/lib/',
    issuerLocator: {
      name: 'react-loadable-ssr-addon',
      reference: 'virtual:ebd7b9259ecd2093755e637a1fd861d705df40e70f047e72ef912a060f6f8b0d7d8b754ceeb087469fa51421c9654efe3ad7ba9d5fbe846f41f5175c740dcb64#npm:0.2.3'
    },
    dependencyName: '@babel/runtime',
    candidates: [ 'react-loadable-ssr-addon', 'react-loadable', 'webpack' ]
  },
  requireStack: [
    '/workspace/Cherry/website/.yarn/$$virtual/react-loadable-ssr-addon-virtual-3a33825c47/0/cache/react-loadable-ssr-addon-npm-0.2.3-26d8c8d441-1.zip/node_modules/react-loadable-ssr-addon/lib/index.js',
    '/workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/lib/commands/build.js',
    '/workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/lib/index.js',
    '/workspace/Cherry/website/.yarn/$$virtual/@docusaurus-core-virtual-ebd7b9259e/0/cache/@docusaurus-core-npm-2.0.0-alpha.58-2f04b5f6f0-1.zip/node_modules/@docusaurus/core/bin/docusaurus.js'
  ]
}
```

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yup

## To Reproduce

Create D2 project with Yarn 2, then run `yarn start`

## Expected behavior

It starts

## Actual Behavior

See stacktrace above

## Your Environment

- Docusaurus version used: v2 beta (latest)
- Environment name and version (e.g. Chrome 78.0.3904.108, Node.js 10.17.0): Node 12.18.1 on Linux
- Operating system and version (desktop or mobile): Gitpod

## Reproducible Demo

(Paste the link to an example repo, including a `siteConfig.js`, and exact instructions to reproduce the issue.)

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

## Motivation

Fixes the project on Yarn v2 (hopefully)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

No significant changes

## Related PRs

Fixes #3003

### CODE REVIEW

**Comment by RDIL** [comment]
cc @slorber

**Comment by slorber** [comment]
weird the CI is stuck and I can't merge :D 

Can you do an empty commit or something to trigger the CI again?

File: packages/docusaurus/package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "react-dev-utils": "^10.2.1",
     "react-helmet": "^6.0.0-beta",
     "react-loadable": "^5.5.0",
-    "react-loadable-ssr-addon": "^0.2.3",
+    "react-loadable-ssr-addon": "^0.3.0",
```
Comment (slorber) [inline packages/docusaurus/package.json:91]:
```suggestion
    "react-loadable-ssr-addon": "^0.3.1",
```

**Comment by RDIL** [comment]
@slorber most likely because you aren’t a member of the organization it was forked to.  I can send an invite if you want.

**Comment by slorber** [comment]
@RDIL I'm a maintainer of this project, normally there's a checkbox on the PR to grant maintainers of the upstream repo write access, it's simpler than inviting me :)

image

**Comment by slorber** [comment]
@RDIL can you open a brand new PR plz?

For some reason the CI does not even start

**Pull request closed without merging**